test/shared/src/main/scala/zio/test/Assertion.scala

```diff
@@ -300,6 +300,18 @@ object Assertion {
       assertion(proj(actual))
     }
 
+  /**
+   * Makes a new assertion that requires an Iterable to have the same elements
+   * than the specified Iterable.
```

Comment:
Minor typo ("than"). Maybe clarify that elements don't necessarily have to be in the same order? Then ready to go!